### PR TITLE
Adobe Animate CC

### PR DESCRIPTION
**Pull request by lukeadams**
Follows the Illustrator cask, but uses proper installer/uninstall stanzas.

### CODE REVIEW

**Comment by lukeadams** [comment]
Fixed all of the Rubocop complaints save for these two
`C: 18:  7: Use 2 spaces for indentation in a hash, relative to the position of the opening brace.`
`C: 20:  5: Indent the right brace the same as the left brace.`

Is this really what it wants?
```ruby
script: {
          executable: staged_path + "Adobe Animate CC 2015/Install.app/Contents/MacOS/Install",
          args:       ['--mode=silent', "--deploymentFile=#{staged_path}/Adobe\ Animate\ CC\ 2015/deploy/uninstall.xml"],
        }
```

File: Casks/adobe-animate-cc.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  url 'http://prodesigntools.com/trials3/AdobeProducts/FLPR/15_1/osx10-64/Animate_15_LS20.dmg',
+      user_agent: :fake,
+      cookies:    { 'MM_TRIALS' => '1234' }
+  name 'Adobe Animate CC'
+  homepage 'https://www.adobe.com/products/animate.html'
+  license :commercial
+
+  installer \
+    script: staged_path + "Adobe Animate CC 2015/Install.app/Contents/MacOS/Install",
+    args:   ['--mode=silent', "--deploymentFile=#{staged_path}/Adobe\ Animate\ CC\ 2015/deploy/en_US_Deployment.xml"]
```
Comment (vitorgalvao) [inline Casks/adobe-animate-cc.rb]: Don’t escape spaces (string is quoted).

File: Casks/adobe-animate-cc.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  uninstall \
+    script: {
+      executable: staged_path + "Adobe Animate CC 2015/Install.app/Contents/MacOS/Install",
+      args:   ['--mode=silent', "--deploymentFile=#{staged_path}/Adobe\ Animate\ CC\ 2015/deploy/uninstall.xml"]
```
Comment (vitorgalvao) [inline Casks/adobe-animate-cc.rb]: Don’t escape spaces (string is quoted).

File: Casks/adobe-animate-cc.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  url 'http://prodesigntools.com/trials3/AdobeProducts/FLPR/15_1/osx10-64/Animate_15_LS20.dmg',
+      user_agent: :fake,
+      cookies:    { 'MM_TRIALS' => '1234' }
+  name 'Adobe Animate CC'
+  homepage 'https://www.adobe.com/products/animate.html'
+  license :commercial
+
+  installer \
+    script: staged_path + "Adobe Animate CC 2015/Install.app/Contents/MacOS/Install",
```
Comment (vitorgalvao) [inline Casks/adobe-animate-cc.rb]: Use `#{staged_path}` instead.

File: Casks/adobe-animate-cc.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    args:   ['--mode=silent', "--deploymentFile=#{staged_path}/Adobe\ Animate\ CC\ 2015/deploy/en_US_Deployment.xml"]
+
+  uninstall \
+    script: {
+      executable: staged_path + "Adobe Animate CC 2015/Install.app/Contents/MacOS/Install",
```
Comment (vitorgalvao) [inline Casks/adobe-animate-cc.rb]: Use `#{staged_path}` instead.

File: Casks/adobe-animate-cc.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 'c4a0affca7c37884471b495a0c242ddb110c8591bc41da91ecab81d12da60590'
+
+  url 'http://prodesigntools.com/trials3/AdobeProducts/FLPR/15_1/osx10-64/Animate_15_LS20.dmg',
+      user_agent: :fake,
+      cookies:    { 'MM_TRIALS' => '1234' }
+  name 'Adobe Animate CC'
+  homepage 'https://www.adobe.com/products/animate.html'
+  license :commercial
+
+  installer \
```
Comment (vitorgalvao) [inline Casks/adobe-animate-cc.rb]: Don’t use `\`, put it on the same line.

File: Casks/adobe-animate-cc.rb [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    script: staged_path + "Adobe Animate CC 2015/Install.app/Contents/MacOS/Install",
+    args:   ['--mode=silent', "--deploymentFile=#{staged_path}/Adobe\ Animate\ CC\ 2015/deploy/en_US_Deployment.xml"]
+
+  uninstall \
```
Comment (vitorgalvao) [inline Casks/adobe-animate-cc.rb]: Don’t use `\`, put it on the same line.

**Comment by vitorgalvao** [comment]
To solve common stylistic issues, run `brew cask style --fix {{your_cask_file}}` prior to submission.

Also, please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by vitorgalvao** [comment]
> Is this really what it wants?

Yes.

**Comment by lukeadams** [comment]
Wow, I had no idea that command existed.
About to squash. Thank you so much for the help!

**Comment by vitorgalvao** [comment]
Thank you for the fast turnout and all the positivity. Always nice to see such good mood from new contributors.

**Pull request merged**